test/test_helper.rb

```diff
@@ -387,4 +381,14 @@ def login_admin
       click_button "Login"
     end
   end
+
+  class ActiveRecord::Base
+    mattr_accessor :shared_connection
+    @@shared_connection = nil
+
+    def self.connection
+      @@shared_connection || ConnectionPool::Wrapper.new(:size => 1) { retrieve_connection }
+    end
+  end
+  ActiveRecord::Base.shared_connection = ActiveRecord::Base.connection
```

Comment:
That also means regular unit/functional tests won't be using the shared connection, right?